help/status-and-availability.md

```diff
@@ -45,6 +55,44 @@ You can set a status emoji, status message, or both.
 
 {end_tabs}
 
+### Remove a status
+
+You can remove a status emoji, status message, or both.
```

Comment:
This always removes both, so this line can probably be omitted so that there is only the header before the instruction block.